scripts/notification_scripts/notification_db_init.js

```diff
@@ -0,0 +1,15 @@
+var rootDBUser = process.env["NOTIF_MONGO_INITDB_ROOT_USERNAME"];
+var rootDBPassword = process.env["NOTIF_MONGO_INITDB_ROOT_PASSWORD"];
```

Comment:
@Edison-Stuart Stay away from shortening descriptive names when possible. It's much clearer to say `NOTIFICATIONS_DBPASSWORD`. Also, the `MONGODB_INITDB_ROOT_USERNAME` and `MONGODB_INITDB_ROOT_PASSWORD` will be the same for the entire mongodb instance, so it will not vary between the init scripts (hence, no need for separate env variables)